FFmpegInterop/AutoResetEvent.h

```diff
@@ -0,0 +1,43 @@
+#include <mutex>
+#include <condition_variable>
+#include <thread>
+#include <stdio.h>
+
+class AutoResetEvent
```

Comment:
I think this is not needed anymore right?

Reply:
Yea, I was thinking about that. Maybe we will need it in the future. But I guess we can add it then.